cmd/operator-ui/vendor/github.com/blackducksoftware/synopsys-operator/pkg/apps/blackduck/interface.go

```diff
@@ -0,0 +1,13 @@
+package blackduck
+
+import (
+"github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+)
+
+type Creater interface {
+	Create(blackduck *v1.Blackduck) error
+	Start(blackduck *v1.Blackduck) error
```

Comment:
what is the difference between create and start?

Reply:
Good catch. I need change this

Reply:
You are looking at the operator-ui vendor directory.  Things should be different in pkg/apps/blackduck/interface.go.  I will take care of the UI in a different PR

Reply:
oops.. in difference, it is not showing the full path :-)